src/components/PostCard/PostTitle/PostTitle.js

```diff
@@ -17,8 +17,10 @@ export default function PostTitle ({
   if (locationObject) {
     if (locationObject.addressNumber !== null && locationObject.addressNumber !== '') {
       generalLocation = `${locationObject.addressNumber} ${locationObject.addressStreet}, ${locationObject.city}, ${locationObject.region}`
-    } else {
+    } else if (locationObject.city !== null && locationObject.city !== '') {
```

Comment:
city both times, should be region

Reply:
You have to check both for a empty string and null. Elsewhere the check is done for both city and region, functionally one just needs to check one of them to get the desired result

Reply:
ah ok, we use isEmpty from lodash in many places